src/app/shared/shared.module.ts

```diff
@@ -21,6 +22,7 @@ import { SafePipe } from './pipes/safe.pipe';
     NgbModule,
     MarkdownModule.forRoot(),
     FhiAngularComponentsModule,
+    FhiTreeViewNavigationComponent,
```

Comment:
I get errors because of this.
![image](https://github.com/user-attachments/assets/7cb44346-e357-4c5f-b1d2-b4d30732c939)

Reply:
Isn't that a strange message given the fact that `FhiTreeViewNavigationComponent` now is standalone?
...and now it doesn't build locally on my machine.